common/dtd/ldmlSupplemental.xsd

```diff
@@ -0,0 +1,2922 @@
+      <xs:attribute name="draft">
+        <xs:simpleType>
+          <xs:restriction base="xs:token">
+            <xs:enumeration value="approved"/>
+            <xs:enumeration value="contributed"/>
+            <xs:enumeration value="provisional"/>
+            <xs:enumeration value="unconfirmed"/>
+            <xs:enumeration value="true"/>
+            <xs:enumeration value="false"/>
+          </xs:restriction>
+        </xs:simpleType>
+      </xs:attribute>
+    </xs:complexType>
+  </xs:element>
+  <!-- @DEPRECATED -->
+  <!-- @DEPRECATED -->
+  <!-- @VALUE -->
+  <!-- @DEPRECATED -->
+  <!-- @METADATA -->
+  <!-- @DEPRECATED -->
+  <xs:element name="territoryCoverage">
+    <xs:complexType>
+      <xs:attribute name="type" type="xs:NMTOKEN"/>
+      <xs:attribute name="values" type="xs:NMTOKENS"/>
+      <xs:attribute name="draft">
+        <xs:simpleType>
+          <xs:restriction base="xs:token">
+            <xs:enumeration value="approved"/>
+            <xs:enumeration value="contributed"/>
+            <xs:enumeration value="provisional"/>
+            <xs:enumeration value="unconfirmed"/>
+            <xs:enumeration value="true"/>
+            <xs:enumeration value="false"/>
+          </xs:restriction>
+        </xs:simpleType>
+      </xs:attribute>
+    </xs:complexType>
+  </xs:element>
+  <!-- @DEPRECATED -->
+  <!-- @DEPRECATED -->
+  <!-- @VALUE -->
+  <!-- @DEPRECATED -->
+  <!-- @METADATA -->
+  <!-- @DEPRECATED -->
+  <xs:element name="currencyCoverage">
+    <xs:complexType>
+      <xs:attribute name="type" type="xs:NMTOKEN"/>
+      <xs:attribute name="values" type="xs:NMTOKENS"/>
+      <xs:attribute name="draft">
+        <xs:simpleType>
+          <xs:restriction base="xs:token">
+            <xs:enumeration value="approved"/>
+            <xs:enumeration value="contributed"/>
+            <xs:enumeration value="provisional"/>
+            <xs:enumeration value="unconfirmed"/>
+            <xs:enumeration value="true"/>
+            <xs:enumeration value="false"/>
+          </xs:restriction>
+        </xs:simpleType>
+      </xs:attribute>
+    </xs:complexType>
+  </xs:element>
+  <!-- @DEPRECATED -->
+  <!-- @DEPRECATED -->
+  <!-- @VALUE -->
+  <!-- @DEPRECATED -->
+  <!-- @METADATA -->
+  <!-- @DEPRECATED -->
+  <xs:element name="timezoneCoverage">
+    <xs:complexType>
+      <xs:attribute name="type" type="xs:NMTOKEN"/>
+      <xs:attribute name="values"/>
+      <xs:attribute name="draft">
+        <xs:simpleType>
+          <xs:restriction base="xs:token">
+            <xs:enumeration value="approved"/>
+            <xs:enumeration value="contributed"/>
+            <xs:enumeration value="provisional"/>
+            <xs:enumeration value="unconfirmed"/>
+            <xs:enumeration value="true"/>
+            <xs:enumeration value="false"/>
+          </xs:restriction>
+        </xs:simpleType>
+      </xs:attribute>
+    </xs:complexType>
+  </xs:element>
+  <!-- @DEPRECATED -->
```

Comment:
Still confused about how these annotations are being used. Is there a mapping from the sequence of annotations to items inside the previous block?

Reply:
The annotations aren't used, I've now removed them.